packages/sdk/src/swap/sdk.ts

```diff
@@ -201,15 +201,14 @@ export class SwapSDK {
   }
 
   async getSwapLimits(): Promise<{
-    minimumSwapAmounts: ChainAssetMap<bigint>;
     maximumSwapAmounts: ChainAssetMap<bigint | null>;
     minimumDepositAmounts: ChainAssetMap<bigint>;
   }> {
     const {
-      swapping: { minimumSwapAmounts, maximumSwapAmounts },
+      swapping: { maximumSwapAmounts },
       ingressEgress: { minimumDepositAmounts },
     } = await this.getStateChainEnvironment();
 
-    return { minimumSwapAmounts, maximumSwapAmounts, minimumDepositAmounts };
+    return { maximumSwapAmounts, minimumDepositAmounts };
```

Comment:
this is a small bc break, how should we handle it?

Reply:
update docs and send some messages in the discord